File: 0980 PROYECTOS/TAREAS/TAREA 2/tarea2.py
import psycopg2 

#Conectar con la base de datos
dbname = 'proyectos'
host = 'localhost'
port = '5432'
user = 'postgres'
password = '202201199'

#Conexión a la base de datos
try:
    with psycopg2.connect(dbname=dbname, user=user, password=password, host=host, port=port) as conn, conn.cursor() as cursor:




        # Insertar datos en la tabla 'redes'
        try:
            cursor.execute("INSERT INTO REDES (NOMBRE, CARNET) VALUES (%s, %s);", ("PABLO", 202201198))
            conn.commit()
            print("Registro insertado correctamente en la base de datos.")
        except Exception as e:
            print(f"Error durante la inserción en la base de datos: {e}")

except Exception as e:
    print(f"Error durante la conexión a la base de datos: {e}")

File: 0980 PROYECTOS/TAREAS/TAREA 2/tarea2.m
pkg load database
%Conectamos con la BD
conn = pq_connect (setdbopts ('dbname',"proyectos",'user',"postgres",'password',"202201199"));

N = pq_exec_params (conn, "INSERT INTO REDES VALUES('PEDRO','202201199');")
M = pq_exec_params (conn, "SELECT * FROM REDES;")
